tests/integration_tests/adapters/interactive_brokers/client/test_client_error.py

```diff
@@ -16,10 +16,13 @@
 import functools
 from unittest.mock import Mock
 
+import pytest
 
+
+@pytest.mark.asyncio
 def test_ib_is_ready_by_notification_1101(ib_client):
     # Arrange
-    ib_client._is_ib_ready.clear()
+    ib_client._is_ib_connected.clear()
```

Comment:
We could probably make a property or method to avoid "private" access, even in tests.